### ISSUE TITLE
Add package to AUR

### ISSUE THREAD

**Issue by PolpOnline** [ndom91/rose-pine-cursor-hyprcursor#1]
### Clear and concise description of the problem

As stated in the title, it would be nice to have this packaged in the AUR

### Suggested solution

Add the package to the AUR

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ndom91** [comment]
Yeah makes sense, I'll put together a PKGBUILD and submit it soon 👍

**Comment by ndom91** [comment]
@PolpOnline alright, it's available under the [rose-pine-cursor-hyprcursor](https://aur.archlinux.org/packages/rose-pine-cursor-hyprcursor) package name. Let me know if you have any troubles with it :crossed_fingers:
